mmv1/third_party/terraform/tests/data_source_google_compute_region_network_endpoint_group_test.go.erb

```diff
@@ -0,0 +1,83 @@
+<% autogen_exception -%>
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceRegionNetworkEndpointGroup(t *testing.T) {
+	// Randomness in instance template
+	skipIfVcr(t)
```

Comment:
Why are we skipping VCR here? I don't see any randomness in the instance template?